x/dex/keeper/cancel_limit_order.go

```diff
@@ -59,20 +59,20 @@ func (k Keeper) ExecuteCancelLimitOrder(
 ) (makerCoinOut, takerCoinOut sdk.Coin, tradePairID *types.TradePairID, error error) {
 	trancheUser, found := k.GetLimitOrderTrancheUser(ctx, callerAddr.String(), trancheKey)
 	if !found {
-		return sdk.Coin{}, sdk.Coin{}, nil, types.ErrActiveLimitOrderNotFound
+		return sdk.Coin{}, sdk.Coin{}, nil, types.ErrValidLimitOrderTrancheNotFound
```

Comment:
according to the [error definition](https://github.com/neutron-org/neutron/blob/bb920dadbfa9a94bd7b4bf95ef86becc590b43ed/x/dex/types/errors.go#L27-L31), it's a wrapped error supposed to be here, isn't it? Here and at the line 75. Just like below at the line 88